douglaseduardo/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script DEFER="DEFER" src="script.js"></script>
+</head>
+
+<body>
+    <section class="top-bar">
+        <div class="input-name-group">
+            <label>Nome:</label>
+            <input class="input-name" type="text">
```

Comment:
Faltou fechar o input, man.
```suggestion
            <input class="input-name" type="text" />
```